src/backend/gl/src/command.rs

```diff
@@ -209,6 +216,16 @@ impl RawCommandBuffer {
         }
     }
 
+    /// Try to take access of the pool and shared memory.
+    pub(crate) fn take_access(&self) -> bool {
+        self.accessible.swap(false, atomic::Ordering::SeqCst)
```

Comment:
Maybe this ordering is better suited https://doc.rust-lang.org/nomicon/atomics.html#acquire-release ?

Reply:
Switched to `Acquire` on `take_access` and `Release` on `release_access`.